ProtectionScan/Features/MainFeature.cs

```diff
@@ -74,6 +82,7 @@ public MainFeature()
             Add(FileOnlyInput);
 #if NETCOREAPP
             Add(JsonInput);
+            Add(NestedInput);
```

Comment:
You can change these two lines to the following in order to ensure that nested can only be used with JSON:

```suggestion
            JsonInput.Add(NestedInput);
            Add(JsonInput);
```